b1ddi/resource_dns_record_test.go

```diff
@@ -109,9 +109,9 @@ func TestAccResourceDnsRecord_full_config(t *testing.T) {
 						comment = "This DNS Record is created by the terraform provider acceptance test"
 						disabled = true
 						name_in_zone = "tf_acc_test_a_record"
-						#options = {
-						#	"create_ptr" = "true"
-						#}
+						options = {
+							"create_ptr" = "true"
```

Comment:
The point of the fix is to allow defining this field as a boolean not as a string.

Reply:
Can `check_rmz` be added in one of the two cases?

Reply:
Done